### ISSUE TITLE
attr member type cannot be inferred from alias

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#938]
**Describe the bug**
When attempting to import an aliased type `ib` from the `attr` package (v19.3.0), Pyright reports that it is an unknown import symbol.  `ib` is the left-hand target in the multiple assignment `ib = attr = attrib` in the `attr` stub file: https://github.com/python-attrs/attrs/blob/19.3.0/src/attr/__init__.pyi#L277.

**To Reproduce**
Install `attrs==19.3.0` and type check the snippet below.

**Expected behavior**
That `ib` would be importable and have the same type as `attrib` and `attr`.

**Screenshots or Code**

```python
# pyright: strict

from attr import (
    attrib, # ok
    attr,   # ok
    ib,     # "ib" is unknown import symbol
)
```

**VS Code extension or command-line**
Extension, 1.1.62

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version. The problem was related to the evaluation of types within an assignment chain.

**Comment by erictraut** [comment]
This is now addressed in version 1.1.63 of Pyright, which I just published. It will also be in the next release of Pylance.

**Comment by layday** [comment]
A variation of this bug was re-introduced in 1.1.72:

```py
# pyright: strict

import attr

attr.ib  # Type of "ib" is unknown
```

**Comment by erictraut** [comment]
Thanks for reporting. You are correct that there was a regression here. It was completely unrelated to the original problem, which is why the regression tests didn't catch it. It was a coincidence that the same symbol was affected. I've introduced a fix, and it will be in the next release.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.73, which I just published.
